Formula/gnutls.rb

```diff
@@ -31,7 +32,8 @@ def install
       --disable-heartbeat-support
       --with-p11-kit
     ]
-
+    ENV["CFLAGS"] = "-I/usr/local/Cellar/unbound/1.9.0/include"
```

Comment:
Are you sure these are needed ? `pkg-config` and homebrew ought to figure this out.

Reply:
Thanks for looking at it. Yes you are correct and I knew about this feature it just wasn't working.
Maybe I got it mixed up.
I added an update that removes the flags and makes the unbound dependency optional.